ch32fun/ch32fun.mk

```diff
@@ -36,6 +36,8 @@ endif
 CFLAGS?=-g -Os -flto -ffunction-sections -fdata-sections -fmessage-length=0 -msmall-data-limit=8
 LDFLAGS+=-Wl,--print-memory-usage -Wl,-Map=$(TARGET).map
 
+GCCVERSIONG14 := $(shell expr `gcc -dumpversion | cut -f1 -d.` \>= 14)
```

Comment:
Should this actually be `GCCVERSIONG14` or should it be `GCCVERSION14`? Why the trailing `G`?